advanced/config.yaml

```diff
@@ -0,0 +1,90 @@
+components:
+  schemas:
+    AdvancedMetrics.Config:
+      description: |-
+        The container for advanced metrics configuration.
+      type: object
+      required: [choice]
+      properties:
+        choice:
+          type: string
+          enum: [events, convergence, latency]
```

Comment:
Shouldn't we get rid of choice here ? since e.g. I should be able to enable both events and convergence metrics in one go.